Scripts/build.swift

```diff
@@ -266,7 +224,7 @@ guard let task = Task(rawValue: rawTask) else {
 }
 
 if task.shouldGenerateXcodeProject {
-    try execute(commandPath: "/usr/bin/swift", arguments: ["package", "generate-xcodeproj", "--output=generated/"])
+    try execute(commandPath: "/usr/bin/xcrun", arguments: ["/usr/bin/swift", "package", "generate-xcodeproj", "--output=generated/"])
```

Comment:
This change is orthogonal, but using `xcrun` means we'll use the `swift` version for the selected Xcode rather than the `swift` version installed on the machine globally.